changelog.txt

```diff
@@ -16,7 +18,7 @@
  * Closing realm on another thread different from where it was created now throws an exception.
  * Realm will now throw a RealmError when Realm's underlying storage engine encounters an unrecoverable error.
  * @Index annotation can also be applied to byte/short/int/long/boolean/Date now.
- * BREAKING CHANGE: Fields with annotation @PrimaryKey are indexed automatically now. Older schemas require a migration.
+ * Fields with annotation @PrimaryKey are indexed automatically now.
```

Comment:
I kinda liked that breaking changes where very visible. Did you guys decide otherwise?